js/main.js

```diff
@@ -66,6 +89,17 @@ document.addEventListener('DOMContentLoaded', function (event) {
   for (let i = 0; i < data.entries.length; i++) {
     const output = renderEntry(data.entries[i]);
     $ul.appendChild(output);
+    const $pencilIcon = output.querySelector('.fa-pencil');
+    $pencilIcon.addEventListener('click', () => {
+      viewSwap('entry-form');
+      const selectedEntry = data.entries[i];
+      data.editing = selectedEntry;
+      $form.elements.title.value = selectedEntry.title;
+      $form.elements['photo-url'].value = selectedEntry.photoUrl;
+      $form.elements.notes.value = selectedEntry.notes;
+      const $entryFormh1 = document.querySelector('.new-entry h1');
+      $entryFormh1.textContent = 'Edit Entry';
+    });
```

Comment:
Couple things here

1. This should not be inside the `DOMContentLoaded` function
2. The event listener should be on your `ul` instead of the pencil icons